pkg/virt-launcher/virtwrap/network/dhcp/dhcp.go

```diff
@@ -100,3 +104,31 @@ func (h *DHCPHandler) ServeDHCP(p dhcp.Packet, msgType dhcp.MessageType, options
 
 	}
 }
+
+func FormClasslessRoutes(routes *[]netlink.Route, routerIP net.IP) (formattedRoutes []byte) {
+	// See RFC4332 for additional information
+	// (https://tools.ietf.org/html/rfc3442)
+	// For example:
+	// 		routes:
+	//				10.0.0.0/8 ,  gateway: 10.1.2.3
+	//              192.168.1/24, gateway: 192.168.2.3
+	//		would result in the following structure:
+	//      []byte{8, 10, 10, 1, 2, 3, 24, 192, 168, 1, 192, 168, 2, 3}
+
+	for _, route := range *routes {
+		if route.Dst == nil {
+			continue
```

Comment:
A route without a destination... So this code basically discards the entire route... Having only skimmed RFC4332, is this the correct thing to do? (the other option being to raise an error). To be clear, I'm not suggesting a change, I'm simply hoping to clarify.

This condition wasn't covered by unit tests. Would it be difficult to add one?

Reply:
> A route without a destination...

The only route without a destination I know is a default gateway, which is set in OptionRouter , not via OptionClasslessRouteFormat. Are there any other cases?

Reply:
Not that I'm aware of. Thanks for clarifying!